nixos/modules/services/monitoring/prometheus/exporters.nix

```diff
@@ -4,8 +4,10 @@ with lib;
 
 let
   cfg = config.services.prometheus.exporters;
+  cfg2 = config.services.prometheus2.exporters;
```

Comment:
Having two sets of exports does not make any sense. Please remove these changes.